modules/oe_theme_contact_forms/oe_theme_contact_forms.module

```diff
@@ -43,11 +43,16 @@ function oe_theme_contact_forms_form_contact_message_form_alter(&$form, FormStat
 
   // Add ecl link classes to the privacy link.
   $privacy_policy = $contact_form->getThirdPartySetting('oe_contact_forms', 'privacy_policy');
-  $privacy_link = Link::fromTextAndUrl('data protection terms', Url::fromUri($privacy_policy, [
+  $privacy_link = Link::fromTextAndUrl(t('data protection terms'), Url::fromUri($privacy_policy, [
     'absolute' => TRUE,
     'attributes' => [
       'target' => '_blank',
-      'class' => ['ecl-u-ml-2xs', 'ecl-link', 'ecl-link--default'],
+      'class' => [
+        'ecl-u-ml-2xs',
+        'ecl-u-mr-2xs',
```

Comment:
We need `ecl-u-mr-2xs` to set space after link when it is used inside string, so without it it looks so:

<img width="463" alt="Screenshot 2021-08-30 at 12 24 39" src="https://user-images.githubusercontent.com/1183951/131317952-162ab9e5-45d0-4808-8be9-b39bbbf7989a.png">